<launch>

    <arg name="source_topic0" default="initialpose0" />
    <arg name="target_topic0" default="/move_base_simple0/goal" />
    <arg name="pub_topic0" default="/planner_node0/car_plan" />
    <arg name="source_topic1" default="initialpose1" />
    <arg name="target_topic1" default="/move_base_simple1/goal" />
    <arg name="pub_topic1" default="/planner_node1/car_plan" />
    <arg name="source_topic2" default="initialpose2" />
    <arg name="target_topic2" default="/move_base_simple2/goal" />
    <arg name="pub_topic2" default="/planner_node2/car_plan" />
    <arg name="source_topic3" default="initialpose3" />
    <arg name="target_topic3" default="/move_base_simple3/goal" />
    <arg name="pub_topic3" default="/planner_node3/car_plan" />
    <arg name="source_topic4" default="initialpose4" />
    <arg name="target_topic4" default="/move_base_simple4/goal" />
    <arg name="pub_topic4" default="/planner_node4/car_plan" />
    <arg name="source_topic5" default="initialpose5" />
    <arg name="target_topic5" default="/move_base_simple5/goal" />
    <arg name="pub_topic5" default="/planner_node5/car_plan" />


    <node pkg="planning_utils" type="planner_node.py" name="planner_node0" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="500" />
        <param name="disc_radius" type="int" value="100" />
        <param name="source_topic" type="string" value="$(arg source_topic0)" />
        <param name="target_topic" type="string" value="$(arg target_topic0)" />
        <param name="pub_topic" type="string" value="$(arg pub_topic0)" />
    </node>
    <node pkg="planning_utils" type="planner_node.py" name="planner_node1" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="500" />
        <param name="disc_radius" type="int" value="100" />
        <param name="source_topic" type="string" value="$(arg source_topic1)" />
        <param name="target_topic" type="string" value="$(arg target_topic1)" />
        <param name="pub_topic" type="string" value="$(arg pub_topic1)" />
    </node>
    <node pkg="planning_utils" type="planner_node.py" name="planner_node2" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="500" />
        <param name="disc_radius" type="int" value="100" />
        <param name="source_topic" type="string" value="$(arg source_topic2)" />
        <param name="target_topic" type="string" value="$(arg target_topic2)" />
        <param name="pub_topic" type="string" value="$(arg pub_topic2)" />
    </node>
    <node pkg="planning_utils" type="planner_node.py" name="planner_node3" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="500" />
        <param name="disc_radius" type="int" value="100" />
        <param name="source_topic" type="string" value="$(arg source_topic3)" />
        <param name="target_topic" type="string" value="$(arg target_topic3)" />
        <param name="pub_topic" type="string" value="$(arg pub_topic3)" />
    </node>
    <node pkg="planning_utils" type="planner_node.py" name="planner_node4" output="screen">
        <param name="static_map" type="string" value="static_map" />
        <param name="halton_points" type="int" value="500" />
        <param name="disc_radius" type="int" value="100" />
        <param name="source_topic" type="string" value="$(arg source_topic4)" />
        <param name="target_topic" type="string" value="$(arg target_topic4)" />
        <param name="pub_topic" type="string" value="$(arg pub_topic4)" />
    </node>
    <!--<node pkg="planning_utils" type="planner_node.py" name="planner_node5" output="screen">-->
        <!--<param name="static_map" type="string" value="static_map" />-->
        <!--<param name="halton_points" type="int" value="500" />-->
        <!--<param name="disc_radius" type="int" value="100" />-->
        <!--<param name="source_topic" type="string" value="$(arg source_topic5)" />-->
        <!--<param name="target_topic" type="string" value="$(arg target_topic5)" />-->
        <!--<param name="pub_topic" type="string" value="$(arg pub_topic5)" />-->
    <!--</node>-->

</launch>
